Core/include/Acts/EventData/MultiTrajectoryHelpers.hpp

```diff
@@ -70,6 +69,7 @@ TrajectoryState trajectoryState(const Acts::MultiTrajectory<traj_t>& multiTraj,
       trajState.measurementVolume.push_back(volume);
       trajState.measurementLayer.push_back(layer);
       trajState.chi2Sum += state.chi2();
+      trajState.NDF += state.calibratedSize();
```

Comment:
I did some investigation locally and found that `NDF` of the trajectory state is different from the `nDoF` in the track. With these changes they are aligned.

Not sure if this should be part of this PR or of another one. If we would merge this change first we should see that the Ambi performance aligns.